ui/tests/acceptance/pki/pki-engine-workflow-test.js

```diff
@@ -444,19 +444,6 @@ module('Acceptance | pki workflow', function (hooks) {
         .dom('[data-test-input="commonName"]')
         .hasValue('Hashicorp Test', 'form prefilled with parent issuer cn');
     });
-
-    test('it returns to the configuration page after submit', async function (assert) {
```

Comment:
this test isn't relevant anymore since we have a tidy route the functionality is different.